cloud/deploys/dev_azure/civiform_config.example.sh

```diff
@@ -0,0 +1,34 @@
+export CIVIFORM_CLOUD_PROVIDER="azure"
+export CIVIFORM_MODE="dev"
+export CIVIC_ENTITY_SHORT_NAME="Sooschester"
+export CIVIC_ENTITY_FULL_NAME="City of Sooschester"
+export CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="CHANGE ME"
+export CIVIC_ENTITY_LOGO_WITH_NAME_URL="https://www.cityofrochester.gov/images/header-logo.png"
+export CIVIC_ENTITY_SMALL_LOGO_URL="https://www.cityofrochester.gov/assets/0/117/8589934986/c6791498-270e-40b1-9b81-aca8a510c1b5.jpg"
+export EMAIL_SENDER="ses"
+export ADMIN_AUTH_PROVIDER="activeDirectory"
+export ADMIN_AUTH_FORMAT="OIDC"
+export RESIDENT_AUTH_PROVIDER="loginRadius"
+export APPLICATION_NAME="CHANGE ME"
+export DOCKER_REPOSITORY_NAME="civiform"
+export DOCKER_USERNAME="civiform"
+export IMAGE_TAG="latest"
+export SENDER_EMAIL_ADDRESS="CHANGE ME"
+export STAGING_HOSTNAME=""
+export STAGING_PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export STAGING_TI_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export STAGING_APPLICANT_NOTIFICATION_MAILING_LIST="CHANGE ME"
+export TERRAFORM_TEMPLATE_DIR="cloud/azure/templates/azure_saml_ses"
+export AZURE_LOCATION="eastus"
+export AZURE_LOG_STORAGE_ACCOUNT_NAME="CHANGE ME"
```

Comment:
For all these azure/aws values - 

Where do you get this from?  Is it someone would have if they are deploying from scratch? 

Some guidance on where to get the values from would be good.

Reply:
The comments from https://github.com/civiform/civiform-deploy/blob/main/civiform_config.example.sh would be very helpful to have here

Reply:
to avoid duplication i added a comment at the top of this file to check this out